language_service/src/completion/tests.rs

```diff
@@ -477,101 +621,75 @@ fn in_block_with_alias() {
 }
 
 #[test]
-fn in_block_from_other_namespace() {
+fn members_of_aliased_namespace() {
     check(
         indoc! {r#"
     namespace Test {
-        operation Bar() : Unit {
-            ↘
+        open FakeStdLib as Alias;
+        operation Foo() : Unit {
+            Alias.↘
         }
-        export Bar;
-    }
-    namespace Other {
-        operation Foo() : Unit {}
-        export Foo;
     }"#},
-        &["Foo"],
+        &["Fake", "Alias.Fake", "Library", "Alias.Library"],
```

Comment:
Might be a bit safer to have it also check against function names that aren't in Alias, like `Foo`, and names that are only in Alias but not elsewhere. In the code as-written, it is a bit ambiguous to me if it is validating the `Fake` or the `Alias.Fake`.